docroot/profiles/custom/cgov_site/modules/custom/cgov_list/config/install/core.entity_view_display.paragraph.cgov_internal_link.ncids_list_item_title_desc.yml

```diff
@@ -0,0 +1,47 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.paragraph.ncids_list_item_title_desc
+    - field.field.paragraph.cgov_internal_link.field_internal_link
+    - field.field.paragraph.cgov_internal_link.field_override_image_promotional
+    - field.field.paragraph.cgov_internal_link.field_override_list_description
+    - field.field.paragraph.cgov_internal_link.field_override_title
+    - paragraphs.paragraphs_type.cgov_internal_link
+  module:
+    - layout_builder
+third_party_settings:
+  layout_builder:
+    enabled: false
+    allow_custom: false
+id: paragraph.cgov_internal_link.ncids_list_item_title_desc
+targetEntityType: paragraph
+bundle: cgov_internal_link
+mode: ncids_list_item_title_desc
+content:
+  field_internal_link:
+    type: entity_reference_entity_view
+    label: hidden
+    settings:
+      view_mode: ncids_list_item_image
```

Comment:
Take a look how lists work. This should not be rendering the internal link as an image because we don't need the image. So this just makes the server do extra work.

This should be `link` like the current lists use.